share/pot/export.sh

```diff
@@ -26,21 +27,63 @@ export-help() {
 _export_pot()
 {
 	# shellcheck disable=SC3043
-	local _pname _dset _snap _tag _dir _file _clvl
+	local _pname _dset _snap _tag _check_tag _prev_tag _prev_snap
+	# shellcheck disable=SC3043
+	local  _dir _file _clvl _meta _origin _origin_pname_snapshot
+	# shellcheck disable=SC3043
+	local _origin_pname _origin_snapshot _origin_tag _highest_version
 	_pname="$1"
 	_snap="$2"
 	_tag="$3"
-	_dir="$4"
-	_clvl="$5"
+	_check_tag="$4"
+	_dir="$5"
+	_clvl="$6"
 	_file="${_dir}/${_pname}_${_tag}.xz"
 	_dset="${POT_ZFS_ROOT}/jails/$_pname"
+        _meta="-"
+
+	_prev_tag=$(zfs get -H :pot.tag "${_dset}" | awk '{ print $3 }')
+	_prev_snap=$(zfs get -H :pot.snap "${_dset}" | awk '{ print $3 }')
+	if [ "$_check_tag" = "YES" ] && \
+	   [ -n "$_prev_tag" ] && [ "$_prev_tag" != "-" ]; then
+		if [ "$_prev_tag" = "$_tag" ] && [ "$_prev_snap" != "$_snap" ]; then
+			_error "Already exported a different snapshot tagged as this version"
+			exit 1
```

Comment:
inside function, please please use `return 1` instead of `exit 1`

Reply:
Makes sense, will change